lib/cdo/rack/response.rb

```diff
@@ -0,0 +1,19 @@
+# Patch Rack::Response with extra utility methods.
```

Comment:
Using a patch instead of a helper-class not essential here, but we already depend heavily on patches within the Rack::Request object and these methods come straight from the v2.0 Rack::Request object (giving the patch an inherently limited life-span as a single-version backport), so I thought it would be more appropriate than usual in this case.